content/guides/references/error-messages.md

```diff
@@ -69,9 +69,8 @@ To include code before your test files, set the
 default, [`supportFile`](/guides/references/configuration#Folders-Files) is set
 to look for one of the following files:
 
-- `cypress/support/index.js`
-- `cypress/support/index.ts`
-- `cypress/support/index.coffee`
+- `cypress/support/component.js`
```

Comment:
The default files have been updated to include more extensions, see https://www.notion.so/cypressdx/Changes-for-10-0-0-1dca213837b84d6ea297c2d12c615f3a#8399c9d90abd416eace7c3f4b2aeb84a

Also, we need to be very clear that for a given testing type, the user can only have one matching support file, multiple matching support files will cause an error. https://www.notion.so/cypressdx/Changes-for-10-0-0-1dca213837b84d6ea297c2d12c615f3a#88cb8241a661447da93a0bf47f86d9f7

Reply:
@cowboy 
Cool. Would you rather note it like we have in notion, e.g`component.supportFile` default value is `cypress/support/component.{js,jsx,ts,tsx}` ...or list out each one to be explicit (like it is now, just a bigger list)?

Reply:
I'm not sure. Can you try a few different approaches and see which looks better?

Reply:
Can this content be moved to the https://deploy-preview-4280--cypress-docs.netlify.app/guides/core-concepts/writing-and-organizing-tests#Plugins-file page and then just link to that? It seems like an error page might not be the best place for it